infra/gcp/lib.sh

```diff
@@ -313,7 +313,7 @@ function empower_group_for_kms() {
 
     local roles=(
         roles/cloudkms.admin
-        roles/cloudkms.cryptoKeyEncrypterDecrypter
+        roles/cloudkms.cryptoOperator
```

Comment:
```suggestion
		roles/cloudkms.cryptoKeyEncrypterDecrypter
        roles/cloudkms.signerVerifier
```

Reply:
@ameukam -- I considered your role suggestion, but opted for Operator, since it's just about equivalent and consolidates a line. Is there a strong preference towards signerVerifier?

Reply:
@justaugustus I think it's better to drop the `cloudkms.locations.*` for something less permissive. I can open a followup PR if it's not enough.